cypress-io/cypress
# fix: launch chrome with --disable-print-preview argument by default

Launch Chrome with `--disable-print-preview` command line flag as noted by @dmtrKovalenko. If we do not do this, Chrome makes a request to an Google Cloudprint endpoint which crashes the runner for some unknown reason.

How to reproduce:

1. Open e2e or CT runner
2. Hit cmd+p.
3. Observe browser crashes.

After this change, you will not get a preview of the page if you hit cmd+p. You can still print the runner webpage if you like, you just won't get a preview. Not really a big deal - this doesn't seem like something users would want to do. If they do, they can still print the page, just without a preview. 

How to test:

1. Open e2e runner (chrome) by `cd packages/runner && yarn cypress:open`
2. Select a spec to run.
3. Hit cmd + p (ctrl + p on windows)
4. Observe the runner does not crash
5. Repeat process for CT runner with (`cd packages/runner-ct && yarn cypress:open`)
